rbac/models.py

```
@@ -0,0 +1,1009 @@
+    )
+    
+    is_active = models.BooleanField(
+        default=True,
+        verbose_name='فعال'
+    )
+    
+    class Meta:
+        db_table = 'user_roles'
+        verbose_name = 'نقش کاربر'
+        verbose_name_plural = 'نقش‌های کاربران'
+        unique_together = [['user', 'role']]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)